editor/src/components/user-bar.tsx

```diff
@@ -196,6 +196,12 @@ const MultiplayerUserBar = React.memo(() => {
     return ownerId === myUser.id
   }, [ownerId, myUser])
 
+  const projectMetadata = useEditorState(
+    Substores.projectServerState,
+    (store) => store.projectServerState.projectData,
```

Comment:
why not 
```suggestion
    (store) => store.projectServerState.projectData?.hasPendingRequests,
```
this way we only need to re-render if the boolean flag flips

Reply:
ah yeah ofc [`ab6abaa` (#5195)](https://github.com/concrete-utopia/utopia/pull/5195/commits/ab6abaad26aa43f4dd5d3817254f027d37e6dca2)